### PR TITLE
Self issue

### PR DESCRIPTION
**Pull request by SimonRichardson**
### Checklist

 - [x] Checked if it requires a [pylibjuju](https://github.com/juju/python-libjuju) change?
 - [x] Added [integration tests](https://github.com/juju/juju/tree/develop/tests) for the PR?
 - [x] Added or updated [doc.go](https://discourse.jujucharms.com/t/readme-in-packages/451) related to packages changed?
 - [x] Do comments answer the question of why design decisions were made?

----

## Description of change

Fix the issue where I typo'd the first argument AFTER I tested, but
before I pushed :(

Anyway, this fixes the issue around that.

## QA steps

```sh
cd acceptancetests
./assess_unregister.py lxd $GOPATH/bin/juju /tmp/artifacts
```

### CODE REVIEW

File: acceptancetests/jujupy/client.py [hunk 1]
```diff
@@ -1142,7 +1142,7 @@ def get_juju_output(self, command, *args, **kwargs):
         """
         model = self._cmd_model(kwargs.get('include_e', True),
                                 kwargs.get('controller', False))
-        return self.get_raw_juju_output(self, command, model, *args, **kwargs)
```
Comment (SimonRichardson) [inline acceptancetests/jujupy/client.py]: I typo'd after I tested :(

File: acceptancetests/jujupy/client.py [hunk 2]
```diff
@@ -1142,7 +1142,7 @@ def get_juju_output(self, command, *args, **kwargs):
...
+        return self.get_raw_juju_output(command, model, *args, **kwargs)
 
     def get_raw_juju_output(self, command, model, *args, **kwargs):
```
Comment (hmlanigan) [inline acceptancetests/jujupy/client.py]: should we put a default value for model in the def of get_raw_juju_output?

File: acceptancetests/jujupy/client.py [hunk 3]
```diff
@@ -1142,7 +1142,7 @@ def get_juju_output(self, command, *args, **kwargs):
...
         """Call a juju command without calling a model for it's values first.
```
Comment (hmlanigan) [inline acceptancetests/jujupy/client.py]: An explanation of why a model is necessary give this line would be helpful.

File: acceptancetests/jujupy/client.py (hunk 2 above)
Comment (SimonRichardson) [reply acceptancetests/jujupy/client.py]: I don't think we need to here, but we can re-evaluate at a later time.

**Comment by SimonRichardson** [comment]
$$merge$$

**Pull request merged**